### PR TITLE
added amp roadshow to events 

### PR DESCRIPTION
**Pull request by keevkeev**
AMP roadshow and its stylings were created. Due to time constraints this week, some of the roadshow contents are not separated Atomically, this will be handled later.

### CODE REVIEW

File: frontend/scss/components/molecules/event.scss [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+*/
+
+@import '_extends.scss';
+@import '_functions.scss';
+@import '_mixins.scss';
+@import '_variables.scss';
+
+@import 'components/atoms/_text.scss';
+@import '../atoms/_color.scss';
```
Comment (matthiasrohmer) [inline frontend/scss/components/molecules/event.scss]: Please use absolute (`@import 'components/atoms/_text.scss';`) imports consistently.

File: frontend/scss/components/molecules/faq.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*
+
+#########################
+### MOLECULE: faq ###
+#########################
+
+*/
+
+@import '../../extends';
```
Comment (matthiasrohmer) [inline frontend/scss/components/molecules/faq.scss]: Please use absolute imports.

File: frontend/templates/views/partials/roadshow-video.j2 [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{% do doc.styles.addCssFile('css/components/molecules/roadshow-video.css') %}
```
Comment (matthiasrohmer) [inline frontend/templates/views/partials/roadshow-video.j2]: As this is possibly never used on any page except the roadshow page, either directly add it to the template or rename to a more generic partial.

File: pages/content/shared/data/roadshow.yaml [hunk 4]
```diff
@@ -0,0 +1,156 @@
+nextstops:
```
Comment (matthiasrohmer) [inline pages/content/shared/data/roadshow.yaml]: There was an update on the `production` branch for the stations, can you check if it still matches those here?

Comment (keevkeev) [reply pages/content/shared/data/roadshow.yaml]: are the added fields to nextstop unwanted?

**Pull request merged**